.github/actions/setup-jimm/action.yml

```diff
@@ -61,12 +61,12 @@ runs:
     - name: Wait for IAM
       uses: nick-fields/retry@v3
       with:
-        timeout_minutes: 10
-        max_attempts: 2 # Retry the wait-for in case it gets stuck and times out. If it fails on the second try it's probably a real error.
+        timeout_minutes: 1
+        max_attempts: 5 # Retry the wait-for with a short timeout in case it gets stuck and times out. If it fails after multiple tries it's probably a real error.
```

Comment:
These changes mean that when the status gets stuck we don't have to wait the full 10 minutes for it to retry, so now if it gets stuck it only has to wait a maximum of 1 min.